cypher_most_mentioned_people.go

```diff
@@ -1,14 +1,13 @@
 package main
 
 import (
-	"fmt"
-
 	"github.com/Financial-Times/neo-model-utils-go/mapper"
 	log "github.com/Sirupsen/logrus"
 	"github.com/jmcvetta/neoism"
 )
-func (pcw CypherDriver) MostMentioned(fromDateEpoch int64, toDateEpoch int64, limit int) (thingList []Thing, found bool, err error) {
-	log.Infof("logging fromDate:%v toDate:%v  limit:%v",fromDateEpoch, toDateEpoch, limit)
+
+func (pcw cypherDriver) MostMentioned(fromDateEpoch int64, toDateEpoch int64, limit int) (thingList []Thing, found bool, err error) {
+	log.Infof("logging fromDate:%v toDate:%v  limit:%v", fromDateEpoch, toDateEpoch, limit)
```

Comment:
This cypher-related code will be modified, as I want to get rid of the unnecessary logs (it's on the TODO list :) )